router.go

```diff
@@ -114,18 +136,26 @@ func (h *RoutableHandler) Enabled(ctx context.Context, l slog.Level) bool {
 //
 //	An error if the underlying handler failed to process the record, nil otherwise
 func (h *RoutableHandler) Handle(ctx context.Context, r slog.Record) error {
+	if h.skipPredicates || h.IsMatch(ctx, r) {
+		return h.handler.Handle(ctx, r)
+	}
+
+	return nil
+}
+
+func (h *RoutableHandler) IsMatch(ctx context.Context, r slog.Record) bool {
```

Comment:
I think `IsMatch` should not be exposed out of the package

Reply:
done